Services/DataBaseService/DbModels/DataBaseContext.cs

```diff
@@ -0,0 +1,40 @@
+﻿using DataBaseService.DbModels;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Reflection;
+
+namespace DataBaseService
+{
+    public class DataBaseContext: DbContext
+    {
+        private readonly IConfiguration configuration;
+
+        public DbSet<DbUserCredential> UsersCredentials { get; set; }
+        
+
+        public DataBaseContext(IConfiguration configuration)
+        {
+            this.configuration = configuration;
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            var connectionStringTradingStation = configuration.GetConnectionString("TradingStationString");
+            try
+            {
+                optionsBuilder.UseSqlServer(connectionStringTradingStation);
+            }
+            catch(Exception e)
```

Comment:
Add comment // TODO Change to custom exception

Reply:
And also : I think you have to throw new exception: to all services understand that something went wrong

Reply:
done